docs/streams/developer-guide/processor-api.html

```diff
@@ -326,14 +326,14 @@ <h2>
                     <p class="last">If the changelog is disabled then the attached state store is no longer fault tolerant and it can&#8217;t have any <a class="reference internal" href="config-streams.html#streams-developer-guide-standby-replicas"><span class="std std-ref">standby replicas</span></a>.</p>
                 </div>
                 <p>Here is an example for enabling fault tolerance, with additional changelog-topic configuration:
-                    You can add any log config from <a class="reference external" href="https://github.com/apache/kafka/blob/1.0/core/src/main/scala/kafka/log/LogConfig.scala#L61">kafka.log.LogConfig</a>.
+                    You can add any log config from <a class="reference external" href="https://github.com/apache/kafka/blob/1.0/core/src/main/scala/kafka/log/LogConfig.scala">kafka.log.LogConfig</a>.
```

Comment:
I just noticed that this references `1.0`. 

We should probably try to pin the branch properly for each release, but since this PR is targeted at `trunk`, I guess we should be pointing to the `trunk` version? (https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/log/LogConfig.scala)

WDYT?

Reply:
Yeah, I've changed, but afterwards I thought if this more reasonable to put the reference to the latest released version? I.e. stick to release tag 2.3.0 https://github.com/apache/kafka/blob/2.3.0/core/src/main/scala/kafka/log/LogConfig.scala . In this case documentation will not reference to the version which not released yet. On the other hand, this link will quickly become obsolete, as people will forget to update it during the release. I concerned which way to go.

Reply:
sounds good to me. I think referencing the "right" branch for each version of the documentation, but realistically, I don't think there's a way to make sure this happens. Pinning to 2.3.0 should be "good enough".

Thanks!